pkg/pillar/agentlog/agentlog.go

```diff
@@ -199,11 +201,37 @@ func handleSignals(log *base.LogObject, agentName string, agentPid int, sigs cha
 				if sigUsr2File != nil {
 					sigUsr2File.Close()
 				}
+				go listenDebug()
 			}
 		}
 	}
 }
 
+func listenDebug() {
+	mux := http.NewServeMux()
+
+	server := &http.Server{
+		Addr:    ":6543",
```

Comment:
I am able to access it from outside (intentionally) until the device is onboarded (not intentionally).

Reply:
(eve firewall drop helps)

Reply:
I don't think we should have this available externally so localhost sounds right.
If we need easy access to this for remote debugging them it would make sense to explore what set of commands we'd need to add to edgeview to be able to get this information. But if it is for lab/testing/tuning (and not production) access, then assuming that the user can ssh or edgeview into a shell in EVE and then run local commands seems reasonable.